packages/ui-components/src/components/test-step-data-viewer/test-step-data-viewer.tsx

```diff
@@ -0,0 +1,74 @@
+import { t } from 'i18next';
+import { useMemo, useState } from 'react';
+import { DynamicToggle } from '../dynamic-toggle/dynamic-toggle';
```

Comment:
I think we should rename it from dynamic-toggle to make it more generic. Maybe toggle-switch?